html-forms/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get">
+      <div class="text">Email</div>
```

Comment:
You want the `Email` text to be within the label. That is to create the functionality of being able to click on the label and activate the input